app.js

```diff
@@ -1,25 +1,22 @@
-const express = require('express')
-const logger = require('morgan')
-const cors = require('cors')
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
 
-const contactsRouter = require('./routes/api/contacts')
+const router = require("./routes/api/routes");
+const app = express();
 
-const app = express()
+const formatsLogger = app.get("env") === "development" ? "dev" : "short";
 
-const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
-
-app.use(logger(formatsLogger))
-app.use(cors())
-app.use(express.json())
-
-app.use('/api/contacts', contactsRouter)
+app.use(morgan(formatsLogger));
+app.use(router);
```

Comment:
Kolejność middleware ma znaczenie, jeśli umieścisz to tutaj to nie będziesz miała dostępu do req.body. Przenieś to na dół zaraz po express.json